src/main/java/duke/command/DukeException.java

```diff
@@ -0,0 +1,10 @@
+package duke.command;
+
+public class DukeException extends Exception{
+    public DukeException(String message) {
+        super(message);
+    }
+    public String toString(){
+        return " Bad command! " + getMessage();
```

Comment:
Avoid using magic string as much as possible. Consider setting `Bad command!` as a constant and use string interpolation to get the desired output.